store/postgres/src/metadata.rs

```diff
@@ -914,3 +914,66 @@ pub(crate) fn revert_subgraph_errors(
 
     check_health(conn, id)
 }
+
+#[test]
```

Comment:
I am not against this, but I thought we put tests that require external components like the store into `tests/`?

Reply:
Actually, leave that here (fear of merge conflicts) - I can always move it later